src/covid_shared/cli_tools.py

```diff
@@ -282,19 +282,49 @@ def setup_directory_structure(output_root: Union[str, Path]):
         if not link_path.is_symlink() and not link_path.exists():
             mkdir(link_path)
 
+    if with_production:
+        production_dir = output_root / paths.PRODUCTION_RUN
+        mkdir(production_dir, exists_ok=True)
+
 
 def mark_best(run_directory: Union[str, Path]):
     """Marks an output directory as the best source of raw input data."""
-    run_directory = Path(run_directory).resolve()
-    best_link = run_directory.parent / paths.BEST_LINK
-    move_link(best_link, run_directory)
+    mark_best_explicit(run_directory, run_directory.parent)
 
 
 def mark_latest(run_directory: Union[str, Path]):
     """Marks an output directory as the best source of raw input data."""
```

Comment:
Shouldn't that be output data

Reply:
yes, hangover from when this was just from etl.